profiles/common/modules/custom/tmgmt_poetry/tmgmt_poetry_mock/src/Mock/PoetryMock.php

```diff
@@ -294,15 +296,22 @@ public static function getDataFromRequest($message) {
    */
   private static function translateRequestContent($content, $language) {
     $decoded_content = base64_decode($content);
-    $xml_content = simplexml_load_string($decoded_content);
+    $dom = new \DOMDocument();
```

Comment:
As i already expressed in ticket NEPT-71 the mock should not mimick the functions in tmgmt_poetry but use them.
I think this ticket is a good opportunity to fix this.

_tmgmt_poetry_replace_job_in_content